App.js

```diff
@@ -17,10 +17,12 @@ export default class App extends Component {
       lat: null,
       long: null,
       userLogged: false,
-      communityMode: false
+      communityMode: false,
+      user: null
     }
 
     this._authorized = this._authorized.bind(this)
+    this._getUser = this._getUser.bind(this)
```

Comment:
I don't understand what these lines are for.
It seems like it doesn't do anything different than the object's method directly, does it?